RocketChat/Rocket.Chat
# fix: Fixed no role assign to manual registered users despite of Default Roles for Users settings

## Proposed changes (including videos or screenshots)
Fixed no role assign to manual registered users despite of Default Roles for Users settings
changes done in apps/meteor/app/lib/server/functions/saveuser.js file and checked it on my local environment and it works fine. Please review this.

https://user-images.githubusercontent.com/67470541/223122109-95c5b7f2-3920-4376-a72d-23723dc1d31c.mp4






## Issue(s)
closes #27736 


## Steps to test or reproduce
1. Define your default roles on Admin UI - Account - Registration - Default Roles for Users
2. Create a manual user through https://yourdomain/admin/users/new
3. Check the role of the user created


## Further comments
